packages/hardhat-core/src/internal/util/global-dir.ts

```diff
@@ -10,12 +11,23 @@ async function generatePaths() {
   return envPaths("hardhat");
 }
 
+function generatePathsSync() {
+  const envPaths: typeof envPathsT = require("env-paths");
+  return envPaths("hardhat");
```

Comment:
When did we change these to `hardhat`? We should be really careful about this.

Reply:
I think this was part of the rename? Anyway, there's a `TODO-HH` below that should remind us to take the migration into consideration.

Reply:
oops, i missed it